core/src/main/scala/org/apache/spark/sql/SnappyContext.scala

```diff
@@ -1035,6 +1035,13 @@ object SnappyContext extends Logging {
         else ExternalClusterMode(sc, sc.master)
       }
     }
+    mode match {
+      case LocalMode(_, _) =>
+        logWarning(s"Initializing SnappyData in cluster mode: $mode")
```

Comment:
Changed now to info level. Earlier had kept it in warning level as per suggestion in SNAP-1678 so that users don't use local mode by mistake. But info will work being the default log level.